Index: jessie-stage2/rootfs/lib/systemd/system/systemd-udevd.service
===================================================================
--- jessie-stage2.orig/rootfs/lib/systemd/system/systemd-udevd.service
+++ jessie-stage2/rootfs/lib/systemd/system/systemd-udevd.service
@@ -24,7 +24,7 @@
 KillMode=mixed
 WatchdogSec=3min
 TasksMax=infinity
-MountFlags=slave
+MountFlags=shared
 MemoryDenyWriteExecute=yes
 RestrictRealtime=yes
 RestrictAddressFamilies=AF_UNIX AF_NETLINK AF_INET AF_INTE6
